Automattic/wp-calypso
# Updating and testing Premium plan bump offer

#### Changes proposed in this Pull Request

* Testing the updated Premium bump offer for qualified Personal plan customers. More info here pNEWy-cIg-p2

#### Testing instructions

* Signup for an account and purchase a Personal plan.
* Put yourself in `control` group for `showPremiumPlanBump` test:
  * After completing payment, verify you're shown the Concierge/Quick Start upsell after purchase.
* Put yourself in `variantShowPlanBump` for `showPremiumPlanBump` test:
  * After completing payment, verify that you are shown the Premium plan bump offer.

Fixes 33-gh-engagement-team
